include/deal.II/lac/scalapack.templates.h

```diff
@@ -271,6 +271,34 @@ extern "C"
            int *      liwork,
            int *      info);
 
+
+  /**
+   * PDTRTRI computes the inverse of a upper or lower triangular
+   * distributed matrix sub( A ) = A(IA:IA+N-1,JA:JA+N-1).
+   *
+   * http://www.netlib.org/scalapack/explore-html/d9/dc0/pdtrtri_8f_source.html
+   * https://www.ibm.com/support/knowledgecenter/SSNR5K_4.2.0/com.ibm.cluster.pessl.v4r2.pssl100.doc/am6gr_lpdtri.htm
+   * https://software.intel.com/en-us/mkl-developer-reference-c-p-trtri
+   */
+  void
+  pdtrtri_(const char *UPLO,
+	   const char *DIAG,
```

Comment:
please run indentation script ` ./contrib/utilities/indent` with clang-format 6.0.0 in path. You can use `./contrib/utilities/compile_clang_format` to compile it.

Reply:
I don't have clang compiler on the machine I am working- only intel and gcc. Is there another approach to run the indentation script?

Reply:
> Is there another approach to run the indentation script?

yes, see above: 

> You can use ./contrib/utilities/compile_clang_format to compile it.

😄

Reply:
I used intel 18.0.1 compiler to compile `./contrib/utilities/compile_clang_format`, and got the following compilation error
```
CMake Error at cmake/modules/CheckAtomic.cmake:53 (message):
  Host compiler appears to require libatomic, but cannot find it.
Call Stack (most recent call first):
  cmake/config-ix.cmake:364 (include)
  CMakeLists.txt:578 (include)
```

Reply:
Can you please check with gcc?

Reply:
Or use the script that simply downloads the executable, assuming you're using linux.

Reply:
gcc/7.3.0 also gave the same compilation error. 

Using `./contrib/utilities/download_clang_format` worked. @bangerth, thanks for the suggestion.